test/tests.js

```diff
@@ -0,0 +1,22 @@
+// IMPORT MODULES under test here:
+// import example from '../src/example.js';
+import isCool from '../isCool.js'
+
+const test = QUnit.test;
+
+test('time to test a function', function(assert) {
+    //Arrange
+    // Set up your parameters and expectations
+    const result = isCool('y');
+    const result2 = isCool('yes');
+    const result3 = isCool('yellow');
+    //Act 
+    // Call the function you're testing and set the result to a const
+
+    //Assert
+    // Make assertions about what is expected valid result
+    assert.equal(true, result);
+    assert.equal(true, result2);
+    assert.equal(true.result3);
```

Comment:
this assertion should not pass due to syntax error.  this is what we call a "false positive".  tests sometimes pass due to weird things.  something to look out for in the future!  The fact that you had a linting error for not using result3 would have been a clue that something was wrong... so check those red squiggles!